release.sh

```diff
@@ -106,6 +106,55 @@ retry() {
 	return $result
 }
 
+# Color codes
+cyan='\033[0;36m'
+yellow='\033[33m'
+red='\033[31m'
+green='\033[0;32m'
+no_color='\033[0m' # Reset colors
```

Comment:
Use tput here it will make the control sequences more portable.

Reply:
TIL! I'll look into your for these, thanks!